its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -80,7 +81,7 @@ public static Location getPluginLocation (String pluginName) {
   public static Build<ScannerForMSBuild> newScanner(Path projectDir) {
     // We need to set the fallback version to run from inside the IDE when the property isn't set
     return ScannerForMSBuild.create(projectDir.toFile())
-      .setScannerVersion(System.getProperty("scannerMsbuild.version", "4.6.1.2049"));
+      .setScannerVersion(System.getProperty("scannerMsbuild.version", "4.8.0.12008"));
```

Comment:
I am not sure this is used anywhere... because the one in the pm.xml is actually used!